python/src/nanoarrow/schema.py

```diff
@@ -945,9 +977,8 @@ def struct(fields, nullable=True) -> Schema:
     ----------
     fields :
         * A dictionary whose keys are field names and values are schema-like objects
-        * An iterable whose items are a schema like object or a two-tuple of the
-          field name and a schema-like object. If a field name is not specified
-          from the tuple, the field name is inherited from the schema-like object.
+        * An iterable whose items are a schema like objects where the field name is
+          inherited from the schema-like object.
```

Comment:
That ability of passing tuples is removed? (don't directly see that in the diff)

Reply:
(Ah, i see now, it's below in the diff)